tests/coverage4gotest.sh

```diff
@@ -21,13 +21,15 @@ function listDeps(){
 }
 
 packages=$(go list ./... | grep -v -E 'vendor|tests|testing')
+echo testing packages: $packages
 
 for package in $packages
 do
 	listDeps $package
 
 #    echo "DEBUG: testing package $package"
-	go test -race -cover -coverprofile=profile.tmp -coverpkg "$deps" $package
+	echo go test -race -v -cover -coverprofile=profile.tmp -coverpkg "$deps" $package
```

Comment:
Do we still need this `echo`?
I remember some of the output was commented to reduce the size of the output

Reply:
I think this is just the benefits for testing log, that we can easily check the command running, or it's hard for us to debug locally when case failed